examples/default/src/screens/apm/NetworkScreen.tsx

```diff
@@ -128,6 +133,7 @@ export const NetworkScreen: React.FC = () => {
             ))}
           </HStack>
         </Section>
+        <ListTile title="HttpUrlConnection" onPress={() => navigation.navigate('HttpScreen')} />
```

Comment:
Let's rename this title to something like "HTTP Screen" since `HttpUrlConnection` is an Android class that's not available in React Native directly.

```suggestion
        <ListTile title="HTTP Screen" onPress={() => navigation.navigate('HttpScreen')} />
```